### PR TITLE
More guarding against KeyErrors

### PR DESCRIPTION
**Pull request by streeter**

### CODE REVIEW

**Comment by streeter** [comment]
Ready for review.

File: iap.py [hunk 1]
```diff
@@ -231,7 +232,7 @@ def validate_production_receipt(decoded_receipt):
 
 
 def validate_debug_receipt(decoded_receipt):
-    if not decoded_receipt['_sandbox']:
```
Comment (meastman) [inline iap.py]: `_sandbox` is set by this library and should always be present.

File: iap.py [hunk 2]
```diff
@@ -215,7 +215,8 @@ def validate_device(decoded_receipt, bundle_ids):
 
 
 def validate_product(decoded_receipt, product_ids):
-    for in_app in decoded_receipt['in_app']:
+    # If there is no products in the receipt, they are all ok
```
Comment (meastman) [inline iap.py]: 'If there are no products in the receipt' ?

Comment (streeter) [inline iap.py]: Oh, right.

File: iap.py (hunk 1 above)
Comment (streeter) [inline iap.py]: Oh right. (meant to comment on this one)

**Comment by meastman** [comment]
:+1:

**Comment by streeter** [comment]
:heart:

**Pull request merged**